docs/products/opensearch/howto/restore-snapshot-opensearch.md

```diff
@@ -0,0 +1,386 @@
+```
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+```bash
+avn service update \
+  --project PROJECT_NAME \
+  --service SERVICE_NAME \
+  -c azure_migration.container="my-container" \
+  -c azure_migration.base_path="snapshots/" \
+  -c azure_migration.account="my-account" \
+  -c azure_migration.key="your-key" \
+  -c azure_migration.snapshot_name="SNAPSHOT_NAME"
+
+```
+
+</TabItem>
+</Tabs>
+
+### S3-compatible services
+
+Register a snapshot repository on an S3-compatible service.
+
+<Tabs groupId="method">
+<TabItem value="api" label="API" default>
+
+```bash
+curl -X PUT "https://SERVICE_NAME.aivencloud.com/_snapshot/my-snapshot-repo" \
+-H "Authorization: Bearer API_TOKEN" \
+-H "Content-Type: application/json" \
+-d '{
+  "type": "s3",
+  "settings": {
+    "bucket": "my-bucket",
+    "region": "us-west-2",
+    "endpoint": "https://my-s3-compatible-service.com",
+    "access_key": "your-access-key",
+    "secret_key": "your-secret-key"
+  }
+}'
+```
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+```bash
+avn service update \
+  --project PROJECT_NAME \
+  --service SERVICE_NAME \
+  -c s3_migration.bucket="my-bucket" \
+  -c s3_migration.region="us-west-2" \
+  -c s3_migration.endpoint="https://my-s3-compatible-service.com" \
+  -c s3_migration.access_key="your-access-key" \
+  -c s3_migration.secret_key="your-secret-key" \
+  -c s3_migration.base_path="snapshots/" \
+  -c s3_migration.snapshot_name="SNAPSHOT_NAME"
+```
+
+</TabItem>
+</Tabs>
+
+## Restore the snapshot
+
+After registering the snapshot repository, restore the snapshot to your Aiven for
+OpenSearch service. This process migrates the data from the repository into your
+Aiven for OpenSearch cluster.
+
+<Tabs groupId="restore-snapshot">
+<TabItem value="api" label="API" default>
+
+```bash
+curl -X POST "https://SERVICE_NAME.aivencloud.com/_snapshot/my-snapshot-repo/SNAPSHOT_NAME/_restore" \
+-H "Authorization: Bearer API_TOKEN" \
+-H "Content-Type: application/json" \
+-d '{
+  "indices": "INDEX_NAME",
+  "ignore_unavailable": true,
+  "include_global_state": false
+}'
+```
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+```bash
+avn service elasticsearch-restore-snapshot \
+  --project PROJECT_NAME \
+  --service SERVICE_NAME \
+  --repository my-snapshot-repo \
+  --snapshot SNAPSHOT_NAME \
+  --indices INDEX_NAME
+
+```
+
+</TabItem>
+</Tabs>
+
+## Monitor the restoration process
+
+Monitor the progress of the snapshot restoration to ensure that everything is
+proceeding as expected.
+
+<Tabs groupId="monitor-restoration">
+<TabItem value="api" label="API" default>
+
+
+```bash
+curl -X GET "https://SERVICE_NAME.aivencloud.com/_snapshot/my-snapshot-repo/SNAPSHOT_NAME/_status" \
+-H "Authorization: Bearer API_TOKEN"
+```
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+
+```bash
+avn service elasticsearch-check-restore-status \
+  --project PROJECT_NAME \
+  --service SERVICE_NAME \
+  --repository my-snapshot-repo \
+  --snapshot SNAPSHOT_NAME
+```
```

Comment:
This command does not exist in aiven-client, this is probably meant to be part of admin CLI which is not accessible by customer.